index.html

```diff
@@ -39,6 +39,190 @@ <h2 class="connect">
             </ul>
         </section>
     </section>
+
+    <!-- Works Section 3 cards-->
+
+    <section class="work-section">
```

Comment:
- [ ] Kindly modify the display for the work section display to grid as requested by the project's requirements.